src/compiler/rust.rs

```diff
@@ -115,7 +115,6 @@ pub struct RustHasher {
 #[derive(Debug, Clone)]
 pub struct RustupProxy {
     proxy_executable: PathBuf,
-    filetime: FileTime,
```

Comment:
@drahnr it seems we don't use filetime elsewhere as well - do you think we can remove similar logic and not fetch  filetimes for the proxies?

Reply:
We do need it, the proxy could be updated, and hence anything is possible :>